mmv1/templates/terraform/examples/healthcare_fhir_store_basic.tf.erb

```diff
@@ -12,6 +12,16 @@ resource "google_healthcare_fhir_store" "default" {
     pubsub_topic = google_pubsub_topic.topic.id
   }
 
+  validation_config {
+    disable_profile_validation        = false
+    disable_required_field_validation = false
+    disable_reference_type_validation = false
+    disable_fhirpath_validation       = false
+    enabled_implementation_guides = [
+      "https://some.url/to-an-implementation-guide"
```

Comment:
It looks like this needs to be a valid implementation guide in order for the test to pass:
> Error: Error creating FhirStore: googleapi: Error 400: could not find ImplementationGuide(s) matching https://some.url/to-an-implementation-guide